### PR TITLE
feat(close) Add promotional close page

### PR DESCRIPTION
**Pull request by quitrk**
@saghul

### CODE REVIEW

File: react/features/app/actions.js [hunk 1]
```diff
@@ -1,5 +1,4 @@
 // @flow
-
```
Comment (saghul) [inline react/features/app/actions.js]: please leave tthis empty line

File: css/main.scss [hunk 2]
```diff
@@ -86,5 +86,6 @@ $flagsImagePath: "../images/";
 @import 'avatar';
 @import 'promotional-footer';
 @import 'chrome-extension-banner';
+@import 'close';
```
Comment (saghul) [inline css/main.scss]: Why make the close page depend on all.css? You can inline the style in the HTML page itself. Or do you need anything else from the all.css file?

Comment (quitrk) [reply css/main.scss]: fair enough

File: react/features/app/actions.js (hunk 1 above)
Comment (quitrk) [reply react/features/app/actions.js]: done

File: css/main.scss (hunk 2 above)
Comment (quitrk) [reply css/main.scss]: done

**Comment by saghul** [comment]
Jenkins add to whitelist.

**Comment by saghul** [comment]
Jenkins please test this please.

**Pull request merged**